internal/scorecard/alpha/tests/bundle_test.go

```diff
@@ -132,6 +132,7 @@ func TestOLMBundle(t *testing.T) {
 	}
 }
 
+//nolint:dupl
```

Comment:
Could we not fix code duplication instead of it? 
shows that the error here is because the both test are using the same cases, so to fix you can, for example, just put the cases on top of the file in order to be a global var and used in both